policyengine_us/parameters/gov/simulation/labor_supply_responses/elasticities/income.yaml

```diff
@@ -1,6 +1,41 @@
-description: Percent change (of the change in disposable income) in labor supply given a 1% change in disposable income.
-values:
-  2020-01-01: 0
-metadata:
-  unit: /1
-  label: income elasticity of labor supply
+all:
```

Comment:
Would there still be value in making a separate `all.yaml` file for the frontend interactions and maintain consistency within the codebase?